test/test_emscripten.py

```diff
@@ -20,7 +20,7 @@
 
 def check_node():
     # cibuildwheel adds a pinned node version to the PATH
-    # check it's in the PATH then, check it's the one that runs pyoodide
+    # check it's in the PATH then, check it's the one that runs pyodide
```

Comment:
While we're fixing typos, maybe also capitalize:
```suggestion
    # check it's in the PATH then, check it's the one that runs Pyodide
```

Reply:
Addressed in 29b5effd8c586d5a2705c43304ba4c4922dbc87c.